src/chrome/content/rules/Lookup.uribl.com.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="Lookup.uribl.com">
+	<target host="lookup.uribl.com" />
+
+	<rule from="^http:" to="https:" />
+
+	<test url="http://lookup.uribl.com/" />
```

Comment:
https://travis-ci.org/EFForg/https-everywhere/jobs/586366798#L328
Remove this test.